package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@clerk/fastify": "^2.0.17",
```

Comment:
**Suggestion: remove this for now, I think the nextjs tooling is sufficient**

Reply:
done